packages/deployer/subtasks/clear-deployments.js

```diff
@@ -11,6 +12,8 @@ subtask(
   SUBTASK_CLEAR_DEPLOYMENTS,
   'Delete all previous deployment data on the current environment'
 ).setAction(async ({ instance }, hre) => {
+  await hre.run(TASK_CLEAN);
```

Comment:
Fly by improvement. Was getting errors sometimes because AST artifacts were not found.